docs/architecture/adr-009-evidence-module.md

```diff
@@ -0,0 +1,205 @@
+# ADR 009: Evidence Module
+
+## Changelog
+
+- 31-07-2019: Initial draft
+
+## Status
+
+Proposed
+
+## Context
+
+In order to support building highly secure, robust and interoperable blockchain
+applications, it is vital for the Cosmos SDK to expose a mechanism in which arbitrary
+evidence can be submitted, evaluated and verified resulting in some agreed upon
+penalty for any equivocations committed by a validator. Furthermore, such a
+mechanism is paramount for any [IBC](https://github.com/cosmos/ics/blob/master/ibc/1_IBC_ARCHITECTURE.md)
```

Comment:
Maybe mention "cross-chain validation" here; I think that's the agreed-upon term.

Reply:
You mean instead of IBC?

Reply:
In addition - I assume this evidence module is intended to be configurable partially to support cross-chain validation in the future?